bower.json

```diff
@@ -22,7 +22,7 @@
     ],
     "dependencies": {
         "underscore": "~1.8.3",
-        "jquery": "~3.1.0",
+        "jquery": "~3.1.1",
```

Comment:
fwiw, `~3.1.0` matches `3.1.1`, and any other patch-level version in the `3.1` series ([SO answer that covers the meaning of `~`](http://stackoverflow.com/a/20412378/10840)). So this PR isn't strictly necessary unless 3.1.1 is required, which I assume it is not.